base/remote/R/fqdn.R

```diff
@@ -19,9 +19,5 @@
 ##' @examples
 ##' fqdn()
 fqdn <- function() {
-  if (Sys.getenv("FQDN") != "") {
-    Sys.getenv("FQDN")
-  } else {
-    system2("hostname", "-f", stdout = TRUE)
-  }
+  as.character(Sys.info()["nodename"])
```

Comment:
If there is a case when the environment variable FQDN should be checked first we could replace this with

```r
  fqdn <- Sys.getenv("FQDN")
  if (fqdn == "") {
    fqdn <- system2("hostname", "-f", stdout = TRUE)
  } 
  if (fqdn == "") {
    fqdn <- as.character(Sys.info()["nodename"])
  }
  return(fqdn)

Reply:
main reason for the FQDN is for an easy way to override the hostname, for example in the case of docker and kubernetes.

Reply:
OK now FQDN will override